### PR TITLE
Laxman

### PR DESCRIPTION
**Pull request by laxmankhanal**
@bishalg @kusalshrestha @saugat135 @vsujan @sanjaymhj @anmol-lf 

Form for adding children have been implemented and form validation is almost completed (date validation not completed)

### CODE REVIEW

File: Vaccination/VaccinationCell.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
+//
+//  VaccinationCell.swift
+//  Vaccination
+//
+//  Created by LF Technology on 2/29/16.
+//  Copyright © 2016 LFTechnology. All rights reserved.
+//
+
```
Comment (bishalg) [inline Vaccination/VaccinationCell.swift]: remove header comment !

File: Vaccination/VaccinationListTVC.swift [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import UIKit
+
+class VaccinationListTVC: UITableView {
+}
```
Comment (bishalg) [inline Vaccination/VaccinationListTVC.swift]: missing end of line

File: Vaccination/VaccinationCell.swift (hunk 1 above)
Comment (laxmankhanal) [inline Vaccination/VaccinationCell.swift]: :ok:

File: Vaccination/VaccinationListTVC.swift (hunk 2 above)
Comment (laxmankhanal) [inline Vaccination/VaccinationListTVC.swift]: :ok:

File: Vaccination/SetFormValue.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
+//
+//  SetFormValue.swift
+//  Vaccination
+//
+//  Created by leapfrog-laxman on 3/8/16.
+//  Copyright © 2016 Leapfrog. All rights reserved.
+//
```
Comment (anmol-lf) [inline Vaccination/SetFormValue.swift]: remove the above comments

Comment (laxmankhanal) [inline Vaccination/SetFormValue.swift]: :ok:

**Pull request closed without merging**